superset/forms.py

```diff
@@ -343,6 +343,13 @@ def __init__(self, viz):
                     "Bottom marging, in pixels, allowing for more room for "
                     "axis labels"),
             }),
+            'page_length': (FreeFormSelectField, {
+                "label": _("Page Length"),
+                "default": 0,
+                "choices": self.choicify([0, 10, 25, 50 , 100, 250, 500]),
```

Comment:
looks like we are using different options here and here: https://github.com/airbnb/superset/pull/1616/files#diff-d46b977e338b1af392d20d572ae7b8b9R1293

should we cap this at 200 entries per page rather than 500? any performance trade offs?